docs/faq.rst

```diff
@@ -130,3 +130,28 @@ How do I shutdown a KSQL environment?
    .. code:: bash
 
        $ confluent destroy
+
+========================================================
+Can KQL connect to a Apache Kafka cluster over SASL_SSL?
+========================================================
+
+Yes! Internally, KSQL uses standard Kafka Consumers and Producers, which you
```

Comment:
suggest rewrite as: `Yes. Internally, KSQL uses standard Kafka consumers and producers. The procedure to securely connect KSQL to Kafka is the same as connecting your app to Kafka. To connect KSQL to a Kafka cluster over SASL_SSL, configure your Kafka consumers and producers to connect to a secure Kafka cluster.`

Reply:
Thanks @joel-hamill. First half of the above is definitely an improvement. Though I'm a bit confused by the second part:

> To connect KSQL to a Kafka cluster over SASL_SSL, configure your Kafka consumers and producers to connect to a secure Kafka cluster.

Not sure that's meaningful / helpful.

Reply:
@big-andy-coates agreed regarding second part - feel free to ignore.